daemon/api_apps_test.go

```diff
@@ -168,6 +178,22 @@ func (s *appsSuite) SetUpTest(c *check.C) {
 	s.AddCleanup(func() { d.Overlord().Stop() })
 	s.AddCleanup(systemd.MockSystemdVersion(237, nil))
 	s.expectAppsAccess()
+
+	xdgRuntimeDir := fmt.Sprintf("%s/%d", dirs.XdgRuntimeDirBase, os.Getuid())
```

Comment:
Not a problem in this PR. But I am surprised that we do not obey `XDG_RUNTIME_DIR` environment variable here.